ci-operator/step-registry/rhcs/e2e/teardown/rhcs-e2e-teardown-commands.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+
+export GOCACHE="/tmp/cache"
+export GOMODCACHE="/tmp/cache"
+export GOPROXY=https://proxy.golang.org
+
+
+# un tar the manifest file
+ls ${SHARED_DIR}
+
+cp -r /root/terraform-provider-rhcs ~/
+# Copy the manifest folder to the shared DIR for below steps share
+cp -r  ~/terraform-provider-rhcs/tests/tf-manifests ${SHARED_DIR}/tf-manifests
+cd ${SHARED_DIR}
+tar -xvf statefiles.tar.gz
+
+ls -R ${SHARED_DIR}/tf-manifests
+
+cd ~/terraform-provider-rhcs
+
+go mod download
+go mod tidy
+go mod vendor
+
+
+RHCS_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
+if [ -z "${RHCS_TOKEN}" ]; then
+    error_exit "missing mandatory variable \$RHCS_TOKEN"
+fi
+export RHCS_TOKEN=${RHCS_TOKEN}
+export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred
+if [ ! -f ${CLUSTER_PROFILE_DIR}/.awscred ];then
+    error_exit "missing mandatory aws credential file ${CLUSTER_PROFILE_DIR}/.awscred"
+fi
```

Comment:
populating TF_VAR_aws_access_key, and TF_VAR_aws_secret_key ?

Reply:
Is there any requirement to populating the aws secrets? 
We just need to set global env `AWS_SHARED_CREDENTIALS_FILE` for terraform provider usage to create resources.

Reply:
There are some terraform yamls with aws_access_key,aws_aws_secret_key (like here: https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/ci/e2e/oidc_configuration/cluster_with_managed_oidc_config/main.tf#L34)

Reply:
The job is not used for the ci under https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/ci/e2e/oidc_configuration/cluster_with_managed_oidc_config
The job is used for the ci run under https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/tests
So no TF_VAR_aws_access_key and TF_VAR_aws_secret_key required.